include/roaring/bitset_util.h

```diff
@@ -16,12 +16,12 @@ static inline void bitset_set_range(uint64_t *bitmap, uint32_t start,
     uint32_t endword = (end - 1) / 64;
     if (firstword == endword) {
         bitmap[firstword] |= ((~UINT64_C(0)) << (start % 64)) &
-                             ((~UINT64_C(0)) >> ((-end) % 64));
+                             ((~UINT64_C(0)) >> ((~end + 1) % 64));
```

Comment:
MSVC doesn't like applying unary minus to unsigned numbers:
warning C4146: unary minus operator applied to unsigned type, result still unsigned